From 11d309701b4a672f77e7dacacf6edefebe14517b Mon Sep 17 00:00:00 2001
From: Vitaly Miniakhmetov <vitalii.miniakhmetov@t-platforms.ru>
Date: Tue, 29 Jun 2021 12:17:19 +0500
Subject: [PATCH 062/106] tp_baikalm_defconfig: Updated CPU config options

- Enabled hotplug CPU
- Disabled `ondemand` governor
- Disabled `conservative` governor
- Enabled `shedutil` governor
- Set `schedutil` to use by default
- Disabled hibernation (research needed)
---
 arch/arm64/configs/tp_baikalm_defconfig | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/configs/tp_baikalm_defconfig b/arch/arm64/configs/tp_baikalm_defconfig
index 0aeae3640c96..4b732e1d8bee 100644
--- a/arch/arm64/configs/tp_baikalm_defconfig
+++ b/arch/arm64/configs/tp_baikalm_defconfig
@@ -35,21 +35,16 @@ CONFIG_ARCH_BAIKAL=y
 CONFIG_ARM64_VA_BITS_48=y
 CONFIG_SCHED_MC=y
 CONFIG_NR_CPUS=8
+CONFIG_HOTPLUG_CPU=y
 CONFIG_HZ_1000=y
 CONFIG_COMPAT=y
-CONFIG_HIBERNATION=y
-CONFIG_CPU_IDLE=y
+# CONFIG_SUSPEND is not set
+CONFIG_PM=y
 CONFIG_CPU_IDLE_GOV_LADDER=y
-CONFIG_CPU_IDLE_GOV_MENU=y
-CONFIG_CPU_IDLE_GOV_TEO=y
 CONFIG_ARM_CPUIDLE=y
-CONFIG_ARM_PSCI_CPUIDLE=y
 CONFIG_CPU_FREQ=y
-CONFIG_CPU_FREQ_STAT=y
-CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_USERSPACE=y
-CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
 CONFIG_CPUFREQ_DT=y
 CONFIG_ACPI=y
 CONFIG_VIRTUALIZATION=y
-- 
2.33.0

